src/collection.ts

```diff
@@ -768,7 +767,14 @@ export default class Collection {
   /**
    * @private
    */
-  async listChangesUntil<T>(at: number): Promise<HistoryEntry<T>[]> {
+  @capable(["history"])
```

Comment:
What is this syntax?

Reply:
https://docs.google.com/document/d/1ikxIP5-RVYq6d_f8lAvf3pKC00W78ueyp-xIZ6q67uU/edit